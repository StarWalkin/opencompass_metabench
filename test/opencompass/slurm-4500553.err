  0%|          | 0/3 [00:00<?, ?it/s]                                       0%|          | 0/3 [00:00<?, ?it/s]slurmstepd: error: *** JOB 4500553 ON gpua053 CANCELLED AT 2024-08-17T18:46:36 ***
