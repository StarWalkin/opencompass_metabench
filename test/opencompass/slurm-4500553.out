/scratch/bcwg/Metabench/anaconda/envs/opencompass/bin/python3
08/17 11:52:48 - OpenCompass - INFO - Loading mmlu_gen: configs/datasets/mmlu/mmlu_gen.py
08/17 11:52:48 - OpenCompass - INFO - Loading hf_llama3_8b_instruct: configs/models/hf_llama/hf_llama3_8b_instruct.py
08/17 11:52:48 - OpenCompass - INFO - Loading hf_qwen2_7b_instruct: configs/models/qwen/hf_qwen2_7b_instruct.py
08/17 11:52:48 - OpenCompass - INFO - Loading hf_yi_1_5_9b_chat: configs/models/yi/hf_yi_1_5_9b_chat.py
08/17 11:52:48 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/17 11:52:48 - OpenCompass - INFO - Current exp folder: outputs/default/20240817_115248
08/17 11:52:48 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/17 11:52:55 - OpenCompass - INFO - Partitioned into 3 tasks.
